t/Util.pm

```diff
@@ -6,14 +6,15 @@ use Digest::MD5 qw(md5_hex);
 use File::Temp qw(tempfile);
 use IO::Socket::INET;
 use IO::Socket::SSL;
+use IO::Poll qw(POLLIN POLLOUT POLLHUP POLLERR);
 use Net::EmptyPort qw(check_port empty_port);
 use POSIX ":sys_wait_h";
 use Path::Tiny;
 use Protocol::HTTP2::Connection;
 use Protocol::HTTP2::Constants;
 use Scope::Guard qw(scope_guard);
 use Test::More;
-use Time::HiRes qw(sleep);
+use Time::HiRes qw(sleep gettimeofday tv_interval);
 
 use base qw(Exporter);
 our @EXPORT = qw(ASSETS_DIR DOC_ROOT bindir run_as_root server_features exec_unittest exec_mruby_unittest spawn_server spawn_h2o spawn_h2o_raw empty_ports create_data_file md5_file prog_exists run_prog openssl_can_negotiate curl_supports_http2 run_with_curl h2get_exists run_with_h2get run_with_h2get_simple one_shot_http_upstream wait_debugger spawn_forked spawn_h2_server);
```

Comment:
FWIW, to export `find_blackhole_ip`, what you need to do is add that name to the list of `@EXPORT`.